general/development/policies/deprecation/scss-deprecation.md

```diff
@@ -0,0 +1,93 @@
+---
+title: SCSS Deprecation
+tags:
+  - Processes
+  - Core development
+  - Deprecation
+  - SCSS
+---
+
+## When should SCSS be deprecated?
+
+Since Moodle 4.4, it's possible to deprecate SCSS styles and classes. This allows us to safely remove SCSS and will help us keep the code cleaner and more organized.
```

Comment:
```suggestion
<Since versions={["4.4"]} issueNumber={"MDL-78334"} />

Since Moodle 4.4, it's possible to deprecate SCSS styles and classes. This allows us to safely remove SCSS and will help us keep the code cleaner and more organized.
```

Reply:
Thanks Sara. Added the "Since" component :grinning: